.travis.yml

```diff
@@ -1,22 +1,43 @@
-sudo: false
-language: scala
-addons:
-  hostname: localhost
-jdk:
-  - oraclejdk8
-scala:
-  - 2.11.12
-  - 2.12.8
-before_install:
-  - docker pull daunnc/openjdk-gdal:2.4.0
+# Required to clean up sbt caches written by container image as root
+sudo: required
+
+# https://github.com/travis-ci/travis-ci/issues/2856
+language: bash
+
 services:
   - docker
+
+before_install:
+  - docker pull quay.io/azavea/openjdk-gdal:2.3.2-jdk8-slim 
+
+env:
+  - SCALA_VERSION=2.11.12
+  - SCALA_VERSION=2.12.8
+
+# These directories are cached to S3 at the end of the build
 cache:
   directories:
-  - $HOME/.ivy2
-  - $HOME/.sbt
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+    - $HOME/.coursier/cache/v1
+
 script:
-  - .travis/build-and-test-docker.sh
+  - scripts/test
+
+before_cache:
```

Comment:
Could we just execute these within the context of the container to get around the permissions issues?